libtextsecure/message_receiver.js

```diff
@@ -1065,12 +1065,26 @@ MessageReceiver.prototype.extend({
       promises.push(this.handleAttachment(attachment));
     }
 
-    if (
-      decrypted.contact &&
-      decrypted.contact.avatar &&
-      decrypted.contact.avatar.avatar
-    ) {
-      promises.push(this.handleAttachment(decrypted.contact.avatar.avatar));
+    if (decrypted.contact && decrypted.contact.length) {
+      const contacts = decrypted.contact;
+
+      for (let i = 0, max = contacts.length; i < max; i += 1) {
+        const contact = contacts[i];
```

Comment:
Could we use `forEach` here or at least `for … of` since we don’t use the index (`i`)?

Reply:
Given that all the code around it uses a full `for` loop, I'm gonna leave it.